javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
     // ... your code goes here
+    if (callback) {
+      this.intervalId = setInterval(callback, 1000);
```

Comment:
Callback is a function that should be executed if it is passed as a parameter, I understand your logic but it should be:
```
if (callback) {
callback()
}
```